<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTasksClass</class>
 <widget class="QWidget" name="DlgTasksClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgTasks</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{
 	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(251,102,102, 220), stop:1 rgba(20,196,188, 230));
	border:none;
	border-radius:10px;
}
#lineEdit
{
	font: 25 12pt '微软雅黑 Light';
	color: rgb(31,31,31);
	padding-left:20px; 
	background-color: rgb(255, 255, 255);
	border:1px solid rgb(20,196,188);
	border-radius:10px;
}
QPushButton
{
	font: 25 12pt '微软雅黑';
	color: rgb(255,255,255);
	background-color: rgb(20,196,188);
	border:none;
	border-radius:10px;
}
QPushButton
{
	font: 25 12pt '微软雅黑';
	color: rgb(255,255,255);
	background-color: rgb(20,196,188);
	border:none;
	border-radius:10px;
}
QPushButton:hover
{
	background-color: rgb(22,218,208);
}
QPushButton:pressed
{
	background-color: rgb(17,171,164);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnConfirm">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
